tests/unit/amo/containers/TestHome.js

```diff
@@ -164,6 +166,36 @@ describe(__filename, () => {
     expect(shelf).toHaveProp('loading', true);
   });
 
+  it('renders a featured themes shelf with the ADDON_TYPE_THEMES_FILTER filter if static theme is enabled', () => {
+    const fakeConfig = getFakeConfig({ enableStaticThemes: true });
+    const root = render({ _config: fakeConfig });
+
+    const shelves = root.find(LandingAddonsCard);
+    const shelf = shelves.find('.Home-FeaturedThemes');
+    expect(shelf).toHaveProp('footerLink', {
```

Comment:
This assertion should also be updated to only look at `query`, right? I'm not sure if the other parts are relevant.

Reply:
yep, you're right. i'll update